### DISCUSSION TITLE
Do the spends in response curve plots represent total spends of a media channel or spends per time period?

### DISCUSSION THREAD

**Discussion by dian-farah** [google/lightweight_mmm#188] [Q&A]
Hi!

I found that the spends in the response curve plots do not match the sum of media spends for the respective channels. Do the spends in the plots represent the total spends or spends per time period? If it is the latter, what is the frequency (e.g. spends per month, spends per week)?

Thank you!

**Comment by steven-struglia** [comment]
It would be whatever frequency your original data is in. As far as I understand it, if you feed in daily media spending data, fit the model to said data, then plot the response curves then those response curves will have an x-axis of daily spend.
